tldw_Server_API/app/core/DB_Management/Kanban_DB.py

```
@@ -45,6 +48,60 @@ def _generate_uuid() -> str:
     return uuid_module.uuid4().hex.lower()
 
 
+def _is_test_context() -> bool:
+    return bool(os.getenv("PYTEST_CURRENT_TEST")) or is_test_mode()
+
+
+def _normalize_db_path(
+    db_path: str,
+    user_id: str,
+    allow_external_db_path: bool
```

Ruff (0.14.10)

Unused function argument: `allow_external_db_path`

(ARG001)